sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/utils/CxxReportSensor.java

```diff
@@ -177,13 +177,16 @@ protected String getStringProperty(String name, String def) {
       directoryScanner.setIncludes(includes.toArray(new String[includes.size()]));
       directoryScanner.scan();
 
-      for (String found : directoryScanner.getIncludedFiles()) {
-        CxxUtils.LOG.debug("Adding report '{}'", found);
+      String [] includeFiles = directoryScanner.getIncludedFiles();
```

Comment:
Not sure how this looks like in the LOG file. If information is redundant we can remove it.